src/redis-benchmark.c

```diff
@@ -1838,7 +1843,7 @@ int main(int argc, const char **argv) {
             const char *argv[21];
             argv[0] = "MSET";
             for (i = 1; i < 21; i += 2) {
-                argv[i] = "key:{tag}:__rand_int__";
+                argv[i] = sdscatprintf(sdsnew(""),"key%s:__rand_int__",tag);
                 argv[i+1] = data;
             }
             len = redisFormatCommandArgv(&cmd,21,argv,NULL);
```

Comment:
this is a bit ugly.
it used to be a pointer to a static string (no need to free), now it's an sds on the heap.
not sure we need to bother freeing these (process is gonna terminate soon, so maybe just add a comment that we don't care?
also the name `argv` clashes with the argument to the function, i suggest to this this opportunity and rename it.

Reply:
@oranagra can you check the latest commit. Addressed the comments above.